physionet-django/project/models.py

```diff
@@ -1340,7 +1342,7 @@ class StorageRequest(BaseInvitation):
         validators=[MaxValueValidator(10240), MinValueValidator(1)])
     responder = models.ForeignKey('user.User', null=True,
         on_delete=models.SET_NULL)
-    response_message = models.CharField(max_length=50, default='', blank=True)
+    response_message = models.CharField(max_length=300, default='', blank=True)
```

Comment:
Maybe bump up to 600? Nice to be able to explain why a request is being declined.